### ISSUE TITLE
Crypto_Triangular_Arbitrage_Bot.py not properly trading

### ISSUE THREAD

**Issue by Roibal** [Roibal/Cryptocurrency-Trading-Bots-Python-Beginner-Advance#7]
I have worked on the **crypto triangular arbitrage bot** many times and documented entire development process of [python based crypto triangular arbitrage bot on Youtube Videos](https://www.youtube.com/playlist?list=PLIFBTFgFpoJ-CNQXcqX9HGgy75aXdNjOZ). 

Currently the triangular arbitrage bot is not working, it will purchase coin 1 and 2, but won't purchase original coin to complete the triangular arbitrage. 

The issue with bids/asks was fixed, currently the issue is getting caught on the requirement of purchasing whole binance coins, due to exchange rate fluctuations etc--although this issue may have been fixed when binance celebrated 1 year anniversary and modified fees. 

Additionally, code will need to be modified for new fee schedule. 

In conclusion, the crypto triangular arbitrage bot is 99.9% functionally, just needs a final polishing to get it to work 100% correctly.

**Comment by Roibal** [comment]
Very Grateful to @Penlo for the progress he has achieved on this issue and helping improve the tri arb bot.

**Comment by Roibal** [comment]
Also , the TriArbTrader.py bot is the best , most highest functioning while the 'Data Collection' is also important if you want to visualize the markets.
